doc/api/events.md

```diff
@@ -133,6 +141,8 @@ Using the `eventEmitter.once()` method, it is possible to register a listener
 that is immediately unregistered after it is called.
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the first example), that means that this is not the correct place for `use strict`.